packages/calling-stateful-client/src/CallClientState.ts

```diff
@@ -215,6 +215,19 @@ export interface RaiseHandCallFeatureState {
   localParticipantRaisedHand?: RaisedHandState;
 }
 
+/* @conditional-compile-remove(ppt-live) */
+/**
+ * State only version of {@link @azure/communication-calling#PPTLiveCallFeature}. {@link StatefulCallClient} will
+ * automatically listen for pptLive on the call and update the state exposed by {@link StatefulCallClient} accordingly.
+ *
+ * @beta
+ */
+export interface PPTLiveCallFeatureState {
+  /**
+   * Proxy of {@link @azure/communication-calling#PPTLiveCallFeature.isActive}.
+   */
+  isActivated: boolean;
```

Comment:
(non blocking comment) I think this used to be `isActive` but there was a discussion on whether this fit the purpose - what was the reason again to change this to `isActivated` instead of keeping as `isActive`?

Reply:
<img width="1075" alt="Screenshot 2024-02-27 at 11 18 17 AM" src="https://github.com/Azure/communication-ui-library/assets/93549644/c8d624cf-8416-446d-beb0-1dd5e2a9844c">

We talked this before. 
But it seems like the isActive is also reasonable.

Let's me change back to isActive for now, as Azure Review board suggest.

Will have the meeting with DevX team to discuss again.

Reply:
Ah yes thanks for that context. Yeah I was unsure if isActive meant the feature was available (from a call capability stand point) or active as in someone was actively sharing a PPTLive. But its the later (meaning someone has started a PPTLive) in which case isActive seems good 👍

The alternative would be to align with video streams and use `isAvailable`:
![image](https://github.com/Azure/communication-ui-library/assets/2684369/78e760fe-c1e6-45b6-8f9f-2617e9f8869f)

Then can later add `isReceiving` is we need more fine grained understanding if the user is able to stream the feed or not

Reply:
Don't change in this PR though, this name should reflect the SDK name and if we did want to align these names and use isAvailable the SDK would need to incorporate that change also